site/examples/check-lists.tsx

```diff
@@ -176,7 +176,7 @@ const CheckListItemElement = ({ attributes, children, element }) => {
         className={css`
           flex: 1;
           opacity: ${checked ? 0.666 : 1};
-          text-decoration: ${checked ? 'none' : 'line-through'};
+          text-decoration: ${!checked ? 'none' : 'line-through'};
```

Comment:
Looks like you already found a bug with your test lol :)

Reply:
yeah,